docs-next/components/Page.tsx

```diff
@@ -23,8 +23,8 @@ export const Page = ({
   const [mobileNavCollapsed, setMobileNavCollapsed] = useState(true);
   const contentRef = useRef<HTMLDivElement | null>(null);
   return (
-    <div className="antialiased pb-24">
-      <div className="py-4 border-b border-gray-200">
+    <div className="pb-24">
```

Comment:
does this wrapping div exist solely for this pb-24 class? 
Is it safe to assume that we can add that class to the body of our __document.tsx file @JedWatson ? 

I didn't move it assuming it needs to be here, but would be nice for us to be able to reduce the amount of wrapping divs in our markup.